### PR TITLE
style(ecs): cleaning code

### PR DESCRIPTION
**Pull request by naezith**

### CODE REVIEW

File: modules/scenes/antara/gaming/scenes/antara.scenes.base.scene.tests.cpp [hunk 1]
```diff
@@ -14,9 +14,13 @@
...
 
-#include <memory>
 #include <doctest/doctest.h>
-#include "antara/gaming/scenes/base.scene.hpp"
+
+//! C++ System Headers
+#include <memory> ///< std::unique_ptr, std::make_unique
+
+//! SDK Headers
+#include "antara/gaming/scenes/base.scene.hpp" ///< base_scene
```
Comment (Milerius) [inline modules/scenes/antara/gaming/scenes/antara.scenes.base.scene.tests.cpp]: don't need to specify base_scene here, we are testing base scene.

File: modules/scenes/antara/gaming/scenes/antara.scenes.manager.tests.cpp [hunk 2]
```diff
@@ -14,9 +14,13 @@
...
 
-#include <memory>
 #include <doctest/doctest.h>
-#include "antara/gaming/scenes/scene.manager.hpp"
+
+//! C++ System Headers
+#include <memory> ///< std::make_unique, std::move
+
+//! SDK Headers
+#include "antara/gaming/scenes/scene.manager.hpp" ///< manager
```
Comment (Milerius) [inline modules/scenes/antara/gaming/scenes/antara.scenes.manager.tests.cpp]: Same remark

File: modules/sfml/antara/gaming/sfml/komodo.intro.scene.cpp [hunk 3]
```diff
@@ -14,16 +14,21 @@
...
+//! Dependencies Headers
+#include <entt/entity/registry.hpp> ///< entt::registry
+#include <entt/signal/dispatcher.hpp> ///< entt::dispatcher
+
+//! SDK Headers
 #include "antara/gaming/sfml/komodo.intro.scene.hpp"
-#include "antara/gaming/sfml/event.play.sound.hpp"
-#include "antara/gaming/audio/component.sound.effect.hpp"
-#include "antara/gaming/graphics/component.sprite.hpp"
+#include "antara/gaming/timer/time.step.hpp" ///< timer::time_step::get_fixed_delta_time
```
Comment (Milerius) [inline modules/sfml/antara/gaming/sfml/komodo.intro.scene.cpp]: Ascii order

File: modules/timer/antara/gaming/timer/fps.hpp [hunk 4]
```diff
@@ -16,13 +16,14 @@
 
 #pragma once
```
Comment (Milerius) [inline modules/timer/antara/gaming/timer/fps.hpp:18]: May be we can rename the file ?

Comment (naezith) [reply modules/timer/antara/gaming/timer/fps.hpp:18]: fps is fine imo, easier to find and understand than tps. file also has fps stuff in it

**Pull request merged**